Tests/Indicators/AverageDirectionalIndexTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using NUnit.Framework;
```

Comment:
Corrected, also added the banner to previous test classes that weren't included.